website/versioned_docs/version-1.0.0-beta2/input.md

```diff
@@ -42,15 +42,94 @@ import { Input } from 'react-native-elements';
 
 ##### This component inherits [all native TextInput props that come with a standard React Native TextInput element](https://facebook.github.io/react-native/docs/textinput.html), along with the following:
 
-| prop                    | default | type                   | description                                      |
-| ----------------------- | ------- | ---------------------- | ------------------------------------------------ |
-| containerStyle          | none    | View style (object)    | styling for Input Component Container (optional) |
-| rightIcon               | none    | React Native Component | displays an icon to the right (optional)         |
-| rightIconContainerStyle | none    | View style (object)    | styling for right Icon Component container       |
-| leftIcon                | none    | React Native Component | displays an icon to the left (optional)          |
-| leftIconContainerStyle  | none    | View style (object)    | styling for left Icon Component container        |
-| inputStyle              | none    | object                 | add styling to input component (optional)        |
-| shake                   | none    | any                    | add shaking effect to input component (optional) |
-| displayError            | none    | bool                   | displays error (optional)                        |
-| errorStyle              | none    | object                 | add styling to error message (optional)          |
-| errorMessage            | none    | string                 | adds error message (optional)                    |
+  * [containerStyle](#containerstyle)
+  * [displayError](#displayerror)
+  * [errorMessage](#errormessage)
+  * [errorStyle](#errorstyle)
+  * [inputStyle](#inputstyle)
+  * [leftIcon](#lefticon)
+  * [leftIconContainerStyle](#lefticoncontainerstyle)
+  * [rightIcon](#righticon)
+  * [rightIconContainerStyle](#righticoncontainerstyle)
+  * [shake](#shake)
+
```

Comment:
Missed the `Reference` header here too

Reply:
I will double check them later this week @iRoachie